stock_analytic/models/stock.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013 Julius Network Solutions
+# Copyright 2015 Clear Corp
+# Copyright 2016 OpenSynergy Indonesia
+# Copyright 2017 ForgeFlow S.L.
+# Copyright 2018 Hibou Corp.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    analytic_account_id = fields.Many2one(
+        string="Analytic Account",
+        comodel_name="account.analytic.account",
+    )
+    analytic_tag_ids = fields.Many2many("account.analytic.tag", string="Analytic Tags")
+
+    def _prepare_account_move_line(
+        self, qty, cost, credit_account_id, debit_account_id, description
+    ):
+        self.ensure_one()
+        res = super(StockMove, self)._prepare_account_move_line(
+            qty, cost, credit_account_id, debit_account_id, description
+        )
+        for line in res:
+            if (
```

Comment:
@alan196 @rousseldenis  It's not really a review, but I'm willing to use this module and I don't catch why there is this condition on the product category's *stock valuation account.account.*
Do you know the reason ?
Thank you very much ! :pray: